cpp/src/db/db_object.cu

```diff
@@ -345,21 +317,15 @@ db_table<idx_t>::db_table() {
 
 template<typename idx_t>
 db_table<idx_t>::~db_table() {
-  for (size_t i = 0; i < columns.size(); i++) {
-    if (columns[i] != nullptr) {
-      ALLOC_FREE_TRY(columns[i], nullptr);
-      columns[i] = nullptr;
-    }
-  }
 }
 
 template<typename idx_t>
 void db_table<idx_t>::addColumn(std::string name) {
   if (columns.size() > size_t { 0 } && column_size > 0)
     throw new std::invalid_argument("Can't add a column to a non-empty table");
```

Comment:
We should use CUGRAPH_EXPECTS and CUGRAPH_FAIL for consistency.

Reply:
Changed to use CUGRAPH_EXPECTS rather than just throwing an exception.